App.tsx

```diff
@@ -54,37 +55,49 @@ function AppNavigator() {
   //updateServerUrl("");
 
   return (
-    <NavigationContainer onReady={hideSplash}>
+    <NavigationContainer
+      onReady={hideSplash}
+      linking={{
+        prefixes: [SCHEME + "://"],
+        config: {
+          screens: {
+            ServerManual: {
+              path: "server",
+              parse: {
+                url: String,
+                username: String,
+                password: String,
+                required: (r) => r === "true",
```

Comment:
Ich würd das `required` nicht in die URL nehmen. Lass uns das an der Existenz des Nutzernamens festmachen. Ich kann mir gerade keinen Use-Case vorstellen.